resources/assets/components/Signup/index.js

```diff
@@ -12,8 +12,7 @@ import Post from '../Post';
 import Empty from '../Empty';
 import Quantity from '../Quantity';
 import TextBlock from '../TextBlock';
-// @TODO: add this back in when we are ready to show history modal.
-// import HistoryModal from '../HistoryModal';
```

Comment:
You can remove these commented lines!

Reply:
wait this is so weird @katiecrane ! It isn't showing up in my text editor so I'm not sure why it is here and all seems up to date...

Reply:
![screen shot 2018-06-20 at 11 00 06 am](https://user-images.githubusercontent.com/9019452/41666713-22cb2808-7479-11e8-8753-6687d493ff51.png)

Reply:
Are you sure you've pushed up the latest/don't have any uncommitted changes?

Reply:
oh wait - that is because they are removed! :)

Reply:
omg WHAT IS WRONG WITH ME

Reply:
I was tricked too!! 😂